sum.c

```diff
@@ -0,0 +1,39 @@
+/* Write a program sum.c given an array of n integers where n > 1, return an array of same size as the input array where every index of the output array is the cumulative sum of elements in the array so far. You can have it so that the array is input by the user.*/
+#include <stdio.h>
+#include <string.h>
+int main(){
```

Comment:
Hm, I'm not sure what your program is exactly doing - it seems to just be printing the number you first input over again. Try and see if you can get this code to do the correct thing, and if you need help, come see me during office hours!